### PR TITLE
Add Code and Test Code

### PR DESCRIPTION
**Pull request by gunnargestsson**
Required by my Integration App that is integrating with a PHP web site.  Intention to put that App onto AppSource for release 15.

### CODE REVIEW

File: Modules/System/Cryptography Management/src/RijnadaelManagementTest.Codeunit.al [hunk 1]
```diff
@@ -0,0 +1,126 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+codeunit 50102 "Rijnadael Management Test"
```
Comment (Hornsyld) [inline Modules/System/Cryptography Management/src/RijnadaelManagementTest.Codeunit.al]:
```suggestion
codeunit 50102 "Rijndael Management Test"
```

File: Modules/System/Cryptography Management/src/RijndaelManagement.Codeunit.al [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    /// <param name=ChiperMode>Represents the cipher mode used in the symmetric algorithm. Valid values: ECB,CBC,CFB,CTS,OFB</param>
+    /// <param name=PaddingMode>Represents the padding mode used in the symmetric algorithm.. Valid values: None,ANSIX923,ISO10126,PKCS7,Zeros</param>    
+    /// </summary>
+    procedure InitRijndaelProvider(EncryptionKey: Text; BlockSize: Integer; ChiperMode: Text; PaddingMode: Text)
+    begin
+        RijndaelManagementImpl.InitRijndaelProvider(EncryptionKey, BlockSize, ChiperMode, PaddingMode);
+    end;
+
+    /// <summary>
+    /// Set new block size value for the RijnadaelManaged class
```
Comment (Hornsyld) [inline Modules/System/Cryptography Management/src/RijndaelManagement.Codeunit.al]:
```suggestion
    /// Set new block size value for the RijndaelManaged class    
```

File: Modules/System/Cryptography Management/src/RijndaelManagement.Codeunit.al [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+    /// Set new block size value for the RijnadaelManaged class    
+    /// <param name=BlockSize>Represents the block size, in bits, of the cryptographic operation.</param>
+    /// </summary>
+    procedure SetBlockSize(BlockSize: Integer)
+    begin
+        RijndaelManagementImpl.SetBlockSize(BlockSize);
+    end;
+
+    /// <summary>
+    /// Set new chiper mode value for the RijnadaelManaged class
```
Comment (Hornsyld) [inline Modules/System/Cryptography Management/src/RijndaelManagement.Codeunit.al]:
```suggestion
    /// Set new chiper mode value for the RijndaelManaged class    
```

File: Modules/System/Cryptography Management/src/RijndaelManagement.Codeunit.al [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+    /// Set new chiper mode value for the RijnadaelManaged class    
+    /// <param name=ChiperMode>Represents the cipher mode used in the symmetric algorithm. Valid values: ECB,CBC,CFB,CTS,OFB</param>
+    /// </summary>
+    procedure SetChiperMode(ChiperMode: Text)
+    begin
+        RijndaelManagementImpl.SetChiperMode(ChiperMode);
+    end;
+
+    /// <summary>
+    /// Set new padding mode value for the RijnadaelManaged class
```
Comment (Hornsyld) [inline Modules/System/Cryptography Management/src/RijndaelManagement.Codeunit.al]:
```suggestion
    /// Set new padding mode value for the RijndaelManaged class    
```

File: Modules/System/Cryptography Management/src/RijndaelManagement.Codeunit.al [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+    /// Set new padding mode value for the RijnadaelManaged class    
+    /// <param name=PaddingMode>Represents the padding mode used in the symmetric algorithm.. Valid values: None,ANSIX923,ISO10126,PKCS7,Zeros</param>    
+    /// </summary>
+    procedure SetPaddingMode(PaddingMode: Text)
+    begin
+        RijndaelManagementImpl.SetPaddingMode(PaddingMode);
+    end;
+
+    /// <summary>
+    /// Set the key and vector for the RijnadaelManaged class
```
Comment (Hornsyld) [inline Modules/System/Cryptography Management/src/RijndaelManagement.Codeunit.al]:
```suggestion
    /// Set the key and vector for the RijndaelManaged class    
```

File: Modules/System/Cryptography Management/src/RijndaelManagement.Codeunit.al [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+    /// <param name=MaxSize>Mazimum Size in bits</param>
+    /// <param name=SkipSize>Skip Size in bits</param>
+    /// </summary>
+    procedure GetLegalBlockSizeValues(var MinSize: Integer; var MaxSize: Integer; var SkipSize: Integer)
+    begin
+        RijndaelManagementImpl.GetLegalBlockSizeValues(MinSize, MaxSize, SkipSize);
+    end;
+
+    /// <summary>
+    /// Get the key and vector from the RijnadaelManaged class
```
Comment (Hornsyld) [inline Modules/System/Cryptography Management/src/RijndaelManagement.Codeunit.al]:
```suggestion
    /// Get the key and vector from the RijndaelManaged class    
```

**Comment by darjoo** [comment]
Hi @gunnargestsson, we're reviewing it internally at the moment and will get it in for the next minor update, 15.1.

**Pull request merged**